org.eclipse.xtext.xbase.lib.tests/src/org/eclipse/xtext/xbase/tests/lib/MapExtensionsTest.java

```diff
@@ -0,0 +1,229 @@
+/*******************************************************************************
+ * Copyright (c) 2012 itemis AG (http://www.itemis.eu) and others.
```

Comment:
You should change to yours here

Reply:
Not clear to me. I have not changed the origin copyright text.
Is the following correct:
```
Copyright (c) 2012 itemis AG (http://www.itemis.eu) and others.
Copyright (c) 2017 Stephane Galland.
```

Reply:
See https://www.eclipse.org/legal/copyrightandlicensenotice.php

Usually the initial writer of a file adds his copyright name.
If you are an employee the copyright owner is usually the employer, otherwise it you. We usually add "and others", which would include any further contribution.